cmd/exporter.go

```diff
@@ -76,10 +77,14 @@ func main() {
 	config.SetEnabledEBPFCgroupID(*enabledEBPFCgroupID)
 	config.SetEnabledHardwareCounterMetrics(*exposeHardwareCounterMetrics)
 	config.SetEnabledGPU(*enableGPU)
+
+	cgroup.SetSliceHandler()
+
 	if modelServerEndpoint != nil {
 		klog.Infof("Initializing the Model Server")
 		config.SetModelServerEndpoint(*modelServerEndpoint)
 	}
+	collector_metric.InitAvailableParam()
```

Comment:
can these two function be merged? The params don't need to be init then set in two functions.

Reply:
yes, updated